app/models/concerns/oregon_digital/video_metadata.rb

```diff
@@ -7,12 +7,13 @@ module VideoMetadata
     # Usage notes and expectations can be found in the Metadata Application Profile:
     # https://docs.google.com/spreadsheets/d/16xBFjmeSsaN0xQrbOpQ_jIOeFZk3ZM9kmB8CU3IhP2c/edit?usp=sharing
 
-    PROPERTIES = %w[height width].freeze
-
     included do
+      initial_properties = properties.keys
       property :height, predicate: ::RDF::Vocab::EXIF.height, multiple: false
-
       property :width, predicate: ::RDF::Vocab::EXIF.width, multiple: false
+      define_singleton_method :video_properties do
+        (properties.select { |_k, v| v.class_name.nil? }.keys - initial_properties)
```

Comment:
The assumption here is that any property with a `class_name` must be a controlled vocabulary.. it would be better if this checked for a class_name and that the class name doesn't include `ControlledVocabularies`?